File: ccif/src/java/other/org/himalay/ccif/CodeVeriFicationCertiFication.java
// Copyright (2013) Krishna C Tripathi. All rights reserved.
// 
// You are not allowed to read/copy/distribute following code without explicit written authorization from Krishna C Tripathi
//
package org.himalay.ccif ;

import java.io.DataInputStream;
import java.io.DataOutputStream;
import java.io.IOException;
import java.lang.Exception;
import java.io.ByteArrayInputStream;
import java.io.PrintStream;
import org.himalay.msgs.runtime.Created;

import org.himalay.msgs.runtime.*;
@Created(date = "Fri Jan 15 01:32:55 EST 2016")

public  class CodeVeriFicationCertiFication extends BinStruct implements PublicBinMsg{




    // members variables

        public CodeVeriFicationCertiFication () // throws Exception
    {
            init();
    }
    
    private void init()
    {
    }

    public int readNoHeader(DataInputStream istream) throws IOException 
    {
    
     return read(istream);
       }
   
    public int read(DataInputStream istream) throws IOException 
    {
    preRead();
        int retVal= 0;

             
        postRead();
        return retVal;
    }


    public int write(DataOutputStream ostream) throws IOException 
    {
    preWrite();
        int retVal= 0;

    
        postWrite();
        return retVal;
    }
    
    public int dump(DumpContext dc) throws IOException 
    {
        dc.indent();dc.getPs().print("CodeVeriFicationCertiFication\n");
    dc.increaseIndent();
        int retVal= 0;
dc.decreaseIndent();
        return retVal;
    }


    

    public int getSize() throws IOException
    {
       DataOutputStream dos= new DataOutputStream(new NullStream());
       return this.write(dos);
    }
    
        

}

// End of code

File: libccif/src/bp/CodeVeriFicationCertiFication.cpp
//**//
//@Created(date = "Mon Feb 01 22:30:13 EST 2016")
// Copyright (2012) Krishna C Tripathi. All rights reserved.
//
// You are not allowed to read/copy/distribute following code without explicit written authorization from Krishna C Tripathi
//
//**//
#include "stdio.h"
#include "CodeVeriFicationCertiFication.h"

#ifdef org_himalay_ccif_CodeVeriFicationCertiFication__USE_SMART_PTR
#define ArrayList(x) SmartPtrList<x>
#define ArrayList_iterator(x) SmartPtrList<x>::iterator
#include "SmartPtrList.h"
#else
#define ArrayList(x) BinMessagePtrList
#define ArrayList_iterator(x) BinMessagePtrList::iterator
#include "BinMessagePtrList.h"
#endif

// Namespace
namespace org {
namespace himalay {
namespace ccif {

using namespace ::org::himalay::msgs::runtime;

// Constructor
CodeVeriFicationCertiFication::CodeVeriFicationCertiFication() {
    references= 0;


}
// Destructor
CodeVeriFicationCertiFication::~CodeVeriFicationCertiFication() {
}

int CodeVeriFicationCertiFication::readNoHeader(DataInputStream&  istream ) { // throws IOException
    int retVal = 0;

    return retVal;
}

int CodeVeriFicationCertiFication::read(DataInputStream&  istream ) { // throws IOException
    int retVal = 0;

    return retVal;
}

int CodeVeriFicationCertiFication::write(DataOutputStream& ostream) { // throws IOException

    int retVal= 0;

    return retVal;
}

int CodeVeriFicationCertiFication::dump(DumpContext& dc) { // throws IOException
    dc.indent();
    dc.getPs().print("CodeVeriFicationCertiFication\n");
    dc.increaseIndent();
    int retVal= 0;
    dc.decreaseIndent();
    return retVal;
}


int CodeVeriFicationCertiFication::getSize() { // throws IOException
    int retVal = 0;

    return retVal;
}

#ifdef org_himalay_ccif_CodeVeriFicationCertiFication__USE_SMART_PTR
void intrusive_ptr_add_ref(org::himalay::ccif::CodeVeriFicationCertiFication* p) {
    p->increaseRef();
};
void intrusive_ptr_release(org::himalay::ccif::CodeVeriFicationCertiFication* p) {
    p->decreaseRef();
};;
#endif


}
}
}

// End of code